ms365_accountcreator/api/account_creation.py

```diff
@@ -30,13 +31,15 @@ def post(self):
         """
         Create a new account
         """
-        try:
-            LOGIC.create_account(** request.get_json())
-            return "", 204
-        except EmailIllegalError:
-            abort(400, "Email illegal")
-        except EmailAlreadyUsedError:
-            abort(400, "Email already used")
-        except GraphApiError as err:
-            abort(500, "Upstream API error")
-            APP_LOGGER.error(err)
+        lang: str = request.headers.get('lang', 'en')
+        with force_locale(lang):
```

Comment:
Is forcing this really necessary? The same pattern (augmenting the accepted languages of the request) as in the default route in routes.py should also work.

Reply:
ok